onnxruntime/core/session/inference_session.cc

```diff
@@ -1020,9 +1024,9 @@ Status InferenceSession::LoadOrtModelWithLoader(std::function<Status()> load_ort
   // Check version mismatch, for now we will only proceed when runtime version matches the model's ort version
   const auto* fbs_ort_model_version = fbs_session->ort_version();
   ORT_RETURN_IF(fbs_ort_model_version == nullptr, "Serialized version info is null. Invalid ORT format model.");
-  ORT_RETURN_IF_NOT(IsOrtModelVersionSupported(fbs_ort_model_version->str()),
-                    "The ORT format model version [", fbs_ort_model_version->str(),
-                    "] is not supported this build ", ORT_VERSION);
+  ORT_RETURN_IF_NOT(IsOrtModelVersionSupported(fbs_ort_model_version->string_view()),
+                    "The ORT format model version [", fbs_ort_model_version->string_view(),
+                    "] is not supported in this build ", ORT_VERSION);
```

Comment:
Short term (for the next couple of releases) would it be helpful to add a doc in the ORT source explaining why we're breaking backwards compatibility and include a link to that in the error message?

Reply:
good idea. should we add it in gh-pages? if we are linking to it I suppose it should be somewhat permanent.

Reply:
neither is permanent. :P 

if it's in /docs the link could be updated after the 1.13 branch is cut to use that tag so it's slightly more permanent. not sure how many users will be affected to have a feel for whether it warrants being in gh-pages and harder to know when it could be removed from gh-pages.

Reply:
added docs/ORT_Format_Update_in_1.13.md